src/components/shareModal/ShareModal.tsx

```diff
@@ -18,6 +18,8 @@ const ShareModal: FC<Props> = ({
   openModal,
   onClickCopyToClipboard,
   stickToBottomMobile = true,
+  showShareToFeed,
```

Comment:
Could have a default value, depending on where this component will be used the most:
- true if business
- false other projects